ui/types/chatbot/chatbot.ts

```diff
@@ -0,0 +1,23 @@
+export interface AIInfoDetailResponse {
```

Comment:
this folder will be removed and the file moved to the new lighthouse folder, right?

Reply:
The AIInfoDetailResponse isn't needed. I've moved `types/ai` folder to `types/lighthouse`